### DISCUSSION TITLE
Why does this download so many files simultaneously, instead of completing one file at a time?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [bpozdena/OneDriveGUI#138] [Q&A]
> ![20231117T193352GMT](https://github.com/bpozdena/OneDriveGUI/assets/42837531/c3c4d01d-14f5-4cb6-8b45-4bfbfb143c4f)

**Comment by bpozdena** [comment]
Can you please specify what are the versions of OneDrive Client and OneDriveGUI ? 

If you are using the alpha versions of client and GUI, than this would be expected for now. The progress reporting will need to be rewritten both in the client and in the GUI.

**Reply by roke-julian-lockhart** [reply]
> Can you please specify what are the versions of OneDrive Client and OneDriveGUI ?

@bpozdena, I'm utilising:

1. [`OneDriveGUI-1.0.3-x86_64.AppImage`](https://github.com/bpozdena/OneDriveGUI/releases/download/v1.0.3/OneDriveGUI-1.0.3-x86_64.AppImage)

1. [`openSUSE:/Factory/standard/x86_64/onedrive-2.4.25-1.2.x86_64.rpm`](https://download.opensuse.org/repositories/openSUSE:/Factory/standard/x86_64/onedrive-2.4.25-1.2.x86_64.rpm)

**Comment by bpozdena** [comment]
Then this is most likely caused when your connection drops during the download process and then resumes even a few seconds later. Unfortunately the client does not report the name of the file it's resuming. 

In any case, fix for this issue is planned for next major versions of the client and the GUI.

**Reply by roke-julian-lockhart** [reply]
> In any case, fix for this issue is planned for next major versions of the client and the GUI.

@bpozdena, is this being tracked in an issue?

**Comment by bpozdena** [comment]
Or maybe the file is for some reason chancing during the download process. The same file name is reported with two different file sizes (35GB and 45GB).

**Reply by roke-julian-lockhart** [reply]
> The same file name is reported with two different file sizes (35GB and 45GB).

@bpozdena, could that be because their paths differ? ~~Where do you see that? Is it depicted by my screenshot? I don't see it.~~ Regardless, do you think this is best filed as a bug, consequently?

**Comment by roke-julian-lockhart** [comment]
As in [`discussions/137#discussioncomment-15039687`](https://github.com/bpozdena/OneDriveGUI/discussions/137#discussioncomment-15039687):

> This discussion isn't resolved, so I don't believe that it should be closed as resolved.

**Comment by bpozdena** [comment]
Can you please provide a screenshot  of the same issue from Onedrive GUI v1.2.2?

**Reply by roke-julian-lockhart** [reply]
@bpozdena, thanks. Shall do when able to. Because it might require some time, I don't mind if you re-close as invalid/unplanned in the meantime. If I can also provide any logs, please advise.
